### PR TITLE
installs ghostscript based on .ghostscript_version file in github pro…

### PR DESCRIPTION
**Pull request by renanigt**

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env bash
 # bin/compile <build-dir>
+GHOSTSCRIPT_VERSION="$(cat $BUILD_DIR/.ghostscript_version)"
 
-echo "-----> Installing Ghostscript 9.22 using build directory $1"
+echo "-----> Installing $GHOSTSCRIPT_VERSION using build directory $1"
```
Comment (RomanMukhin) [inline bin/compile:5]: Installing Ghostscript $GHOSTSCRIPT_VERSION

Comment (renanigt) [reply bin/compile:5]: I've put `GHOSTSCRIPT_VERSION="$(cat $BUILD_DIR/.ghostscript_version)"` before `echo`.
It won't work?
@RomanMukhin

Comment (renanigt) [reply bin/compile:5]: Anyway I can change it.

Comment (RomanMukhin) [reply bin/compile:5]: @renanigt you are right, it wouldn't :)

File: bin/compile [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 BUILD_DIR=$1
-PACKAGE="https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs922/ghostscript-9.22-linux-x86_64.tgz"
-BINARY="ghostscript-9.22-linux-x86_64/gs-922-linux-x86_64"
 LOCATION="$BUILD_DIR/vendor/gs"
+PACKAGE="https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs922/ghostscript-$GHOSTSCRIPT_VERSION-linux-x86_64.tgz"
```
Comment (RomanMukhin) [inline bin/compile]: @renanigt gs922, 922 is hardcoded here

Comment (renanigt) [reply bin/compile]: done.

**Comment by renanigt** [comment]
@RomanMukhin done. something else?

**Review by RomanMukhin — APPROVED** [review]

**Pull request merged**